pkg/ccl/sqlccl/csv.go

```diff
@@ -686,7 +686,7 @@ func importPlanHook(
 			// TODO(dt): verify db exists
 		}
 
-		comma := ','
+		var comma rune
```

Comment:
correct, was previously initializing the override to the default, now leaving it zero which gets changed to default in the reader.